opensearch/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG TAG=2.11.0
```

Comment:
I think we can just go with 'latest'?

Reply:
But you have to keep updating this file.  We only care about >= 2.11, not == 2.11.  People can modify this locally so no big deal.